### ISSUE TITLE
sorting community solutions doesn't work

### ISSUE THREAD

**Issue by catalin560** [exercism/exercism#5092]
Hello!
Attempting to sort by any order seems to do nothing at the moment when visiting the community solutions page of any exercise

**Comment by iHiD** [comment]
Yes! We seem to have broken this with the upgrade to Rails 6 earlier. I just noticed the same. We'll get it fixed in the next 24hrs for you!
